Doc/library/mmap.rst

```diff
@@ -277,6 +277,8 @@ To map anonymous memory, -1 should be passed as the fileno along with the length
       pagefile) will silently create a new map with the original data copied over
       up to the length of the new size.
 
+      .. availability:: Linux, Windows
```

Comment:
It is also available on other platforms (e.g. FreeBSD).

See how this is documented for `madvise()`.